### DISCUSSION TITLE
NextJS sandboxes always have internal server errors

### DISCUSSION THREAD

**Discussion by sarahk** [codesandbox/codesandbox-client#6603] [Q&A] [answered]
I've created a few sandboxes successfully with react, but when I try to use nextjs the output frame just says "internal server error"

example: https://codesandbox.io/s/sad-mcnulty-923zr0?file=/pages/index.js

Terminal, console and problems are clear. 

<img width="288" alt="image" src="https://user-images.githubusercontent.com/666147/161954442-5a80276f-42b9-4d44-b538-e7f918011659.png">

Where am I going wrong?

**Comment by JamesACS** [comment, marked as answer]
Hey @sarahk - I've just looked and this is loading now. Did you resolve the issue?

**Comment by sarahk** [comment]
Working for me now, created a new sandbox successfully. Had tried over a few weeks unsuccessfully, so very pleased whatever was glitching for me is sorted. Thanks for the reply.
